README.md

```diff
@@ -4,50 +4,55 @@ Fourier Accelerated Nodal Solvers (FANS) is an FFT-based homogenization solver d
 
 <img src="docs/images/FANS_example.png" alt="Example Image" width="400" height="300">
 
-## Table of Contents
+## Dependencies
 
-- [Installation](#installation)
-- [Input File Format](#input-file-format)
-- [Examples](#examples)
-- [Acknowledgements](#acknowledgements)
+FANS has the following dependencies:
 
-## Installation
-
-### Prerequisites
-
-Before proceeding with the installation, ensure that your system has the necessary dependencies. The prerequisites of FANS can be installed using Spack for a streamlined setup on high-performance computing systems, or through traditional package management for personal use.
-
-### Traditional Installation
-
-If you're setting up FANS on a personal computer or in a non-HPC environment, follow these instructions:
-
-Please ensure you have the following dependencies installed on your system:
-
-- CMake (version 3.0 or higher)
+- A C++ compiler (e.g. GCC)
+- CMake (version 3.0 or higher) (+ GNU file utility for creating .deb packages)
+- Git (for cloning this repo)
 - MPI (mpicc and mpic++)
 - HDF5 with parallel support
 - Eigen3
 - FFTW3 with MPI support
 
-Specifically, to run FANS, you at least need the following packages:
+### Installing dependencies
+
+We recommend installing the dependencies using a package manager. For example, using `apt`, the following commands are run:
 
 ```bash
-openmpi-bin libc6 libfftw3-double3 libfftw3-mpi3 libgcc-s1 libgomp1 libhdf5-103 libopenmpi3 libstdc++6
+apt-get install \
+    libhdf5-dev \
+    libopenmpi-dev \
+    libeigen3-dev \
+    libfftw3-dev \
+    libfftw3-mpi-dev
 ```
 
-To build fans, you additionally need these packages:
+Also we recommend to install these commonly used tools with FANS and setup a virtualenv for the `h52xdmf.py` script:
 
 ```bash
-libhdf5-dev libopenmpi-dev libeigen3-dev libfftw3-dev libfftw3-mpi-dev
+apt-get install \
+    time \
+    htop \
+    python3 \
+    python3-pip \
+    python3-venv \
+    python-is-python3 \
+    python3-dev
+
+python -m venv ~/venvs/FANS
+source ~/venvs/FANS/bin/activate
+python -m pip install h5py lxml
```

Comment:
```suggestion
python -m pip install numpy scipy h5py lxml plotly nbformat notebook jupyterlab ipykernel
```

Reply:
Ack.. honestly would be nice if we could have a pyproject.toml file even...

Reply:
Maybe let's put that in another PR, or/and open an issue to discuss details there? I'm not super experienced with python packages etc.